### ISSUE TITLE
support viewing/editing Border polygon or rectangle

### ISSUE THREAD

**Issue by bertsky** [OCR4all/LAREX#194]
Page frame detection is not always trivial: textual noise from facing pages, structured background, non-orthogonal sight lines due to skew or warp – scans (or worse: photos) can offer many challenges.

But a correct `Border` is often necessary for downstream processing like separator detection or region segmentation.

As an annotation tool, IMO Larex should both _show_ the annotated page frame (if any) and allow _editing_ it manually.

**Comment by maxnth** [comment]
If I understand correctly, treating the result of the already existing `RoI` tool (see below) as `Border` and reading/writing it to the corresponding part in the generated PAGE XML should provide this feature, correct?
![image](https://user-images.githubusercontent.com/33344081/85726681-9d91a780-b6f6-11ea-8ec4-4dea9bea8039.png)

**Comment by bertsky** [comment]
> f I understand correctly, treating the result of the already existing `RoI` tool (see below) as `Border` and reading/writing it to the corresponding part in the generated PAGE XML should provide this feature, correct?

Yes, but is the RoI tool truly polygonal (as `Border` in PAGE), or merely rectangular?

Also, IIUC the RoI tool is not available in `edit` mode (but only `segment` mode), right?

**Comment by maxnth** [comment]
>Yes, but is the RoI tool truly polygonal (as Border in PAGE), or merely rectangular?

At the moment it's merely rectangular as well as the `ignore rectangle` but adding an `ignore polygon` shouldn't break anything, so adding this might make sense.

>Also, IIUC the RoI tool is not available in edit mode (but only segment mode), right?

Yes.
